integration_test/update_test.go

```diff
@@ -49,6 +49,50 @@ func TestKrewUpdate(t *testing.T) {
 	}
 }
 
+func TestKrewUpdateMultipleIndexes(t *testing.T) {
+	skipShort(t)
+	test, cleanup := NewTest(t)
+	defer cleanup()
+
+	test = test.WithEnv(constants.EnableMultiIndexSwitch, 1).WithIndex()
+	os.Setenv(constants.EnableMultiIndexSwitch, "1")
+	defer os.Unsetenv(constants.EnableMultiIndexSwitch)
+
+	paths := environment.NewPaths(test.Root())
+	test.Krew("index", "add", "foo", paths.IndexPath(constants.DefaultIndexName)).RunOrFail()
+	if err := os.RemoveAll(paths.IndexPluginsPath("foo")); err != nil {
+		t.Errorf("error removing plugins directory from index: %s", err)
+	}
+	test.Krew("update").RunOrFailOutput()
+	out, err := ioutil.ReadDir(paths.IndexPluginsPath("foo"))
+	if err != nil {
+		t.Errorf("error reading plugins directory: %s", err)
+	}
+	if len(out) == 0 {
+		t.Error("no plugins in index foo after update")
+	}
```

Comment:
shouldn't you be also checking what's in `default` index now?
also you can use something like `krew search` here to find plugins in all indexes.

Reply:
Ya good call :+1:

Reply:
I think it should be enough to look for two plugin names in the output. For example, `ctx` and `foo/ctx`.